zebra-consensus/src/primitives/halo2/tests.rs

```diff
@@ -50,7 +50,7 @@ fn generate_test_vectors() {
 
             for _ in 0..num_recipients {
                 builder
-                    .add_output(None, recipient, NoteValue::from_raw(note_value), None)
+                    .add_output(None, recipient, NoteValue::from_raw(note_value), [0; 512])
```

Comment:
This will change the test vectors, I think, because `None` meant 0xF6 followed by 511 zero bytes.

Reply:
Thank you! Fixed in https://github.com/ZcashFoundation/zebra/pull/9300/commits/15b1e2d5685d516c329fbff6c64414af107e9678.